recipes/cassandra-cpp-driver/all/conandata.yml

```diff
@@ -0,0 +1,9 @@
+sources:
+  "2.15.3":
+    url: "https://github.com/datastax/cpp-driver/archive/2.15.3.zip"
+    sha256: "494b35418f1eaa86d80572a4254b7fae88a1341dcda83788ed038ce4f39117cb"
+patches:
+  "2.15.3":
+    - base_path: "source_subfolder"
+      patch_file: "patches/2.15.3/fix-rapidjson.patch"
+      patch_file: "patches/2.15.3/fix-cmake.patch"
```

Comment:
```suggestion
    - base_path: "source_subfolder"
      patch_file: "patches/2.15.3/fix-rapidjson.patch"
    - base_path: "source_subfolder"
      patch_file: "patches/2.15.3/fix-cmake.patch"
```

Reply:
[The correct order of patches](https://github.com/conan-io/conan-center-index/pull/3110/commits/3b91907e1bf34c3304e28d9d7abe973e933f3692)

Reply:
[Look here](https://github.com/conan-io/conan-center-index/pull/3110#discussion_r516520754)